assets/chezmoi.io/docs/user-guide/include-files-from-elsewhere.md

```diff
@@ -73,6 +73,11 @@ When using Oh My Zsh, make sure you disable auto-updates by setting
 `~/.oh-my-zsh` directory to drift out of sync with chezmoi's source state. To
 update Oh My Zsh and its plugins, refresh the downloaded archives.
 
+!!! note
+    If your external dependency uses cache files inside directory is unpacked the 
+    chezmoi will report changed files when try to use `chezmoi apply`. 
+    To remove this please use [`.chezmoiignore`](/reference/special-files-and-directories/chezmoiignore/) file.
+
```

Comment:
You have trailing whitespace on your lines, but I would reword this in any case:

```markdown
!!! note

    If your external dependency target directory can contain cache files that are
    added during normal use, chezmoi will report that files have changed on
    `chezmoi apply`. To avoid this, add the cache directory to your
    [`.chezmoiignore`](/reference/special-files-and-directories/chezmoiignore/)
    file.

    For example, Oh My Zsh may cache completions in `.oh-my-zsh/cache/completions/`,
    which should be added to your `.chezmoiignore` file.